src/opm/parser/eclipse/EclipseState/Grid/FieldProps.hpp

```diff
@@ -132,7 +134,7 @@ class FieldProps {
 
 
     FieldProps(const Deck& deck, const EclipseGrid& grid, const TableManager& table_arg);
-    void reset_grid(const EclipseGrid& grid);
+    void reset_actnum(const std::vector<int>& actnum);
```

Comment:
I know you do not like comments, but I see no other way to communicate the (unchecked) prerequisites on actnum.
The way I read the code the new active cells have to be a subset of the old ones. If not the code will produces rubbish without yelling.

Reply:
The new ACTNUM must satisfy:

1. Same size as the original.
2. It is not possible to actviate cells - i.e. 0 &rightarrow; 1

Both of these are checked - but they were already checked in the original code and might not show up in the diff. I'll add a comment.